Add colour differentiation and examples for more commands

moisterrific opened the pull request:
<!-- Warning: If you create a pull request and wish to remain anonymous, you are highly advised to use Tails (https://tails.boum.org/) or a fresh git environment. We will *not* be able to help with anonymization after your pull request has been created. -->

<!-- Warning: If you do not update the changelog, your pull request will be ignored and eventually closed, without comment, without any support, and without any opinion or interaction from the TShock team. This is your only warning. -->

https://github.com/Pryaxis/TShock/issues/2418

moisterrific committed:
add colours for GetVersion

testing something

moisterrific committed:
add colour for ListConnectedPlayers

also various other minor improvements

moisterrific committed:
Update CHANGELOG.md

moisterrific committed:
add colours to Mute

also added silent support

moisterrific committed:
Merge branch 'general-devel' of https://github.com/moisterrific/TShock into general-devel

moisterrific committed:
add colours to /w, /wallow, and /r

and various minor improvements

moisterrific committed:
add colours to annoy

and add silent support

moisterrific committed:
add colour and example for Rocket

along with other minor improvements

moisterrific committed:
add colour msg for Rocket and example

we do a lil refactoring, also added a cyan pastel coloured hex code and made various other little improvements

moisterrific commented:
correction: https://github.com/Pryaxis/TShock/pull/2419/commits/e88ae85683673a2f298503315ce67d18a9d330bd is for Firework command, not Rocket

moisterrific committed:
add examples for Clear + minor refactor

moisterrific committed:
add colours and examples for /kill

also a bit of refactoring and other small improvements

moisterrific committed:
Fix error msg error in /kill

moisterrific committed:
add colours + usage example for /butcher

also added silent support

hakusaro commented on TShockAPI/Commands.cs:
```diff
@@ -5578,94 +5663,115 @@ private static void Clear(CommandArgs args)
 							{
 								Main.item[i].active = false;
-								TSPlayer.All.SendData(PacketTypes.ItemDrop, "", i);
+								everyone.SendData(PacketTypes.ItemDrop, "", i);
```
I get the feeling you want to be a ruby developer but don't know it

moisterrific replied:
idk what u mean lmao

moisterrific committed:
add example for /heal and custom heal amount

code is mostly based from /slap, now users will be able to heal a target player by a custom HP amount. I deliberately changed the static 600 HP amount to statLifeMax2 so it would use whatever max HP the target player has, this should make it work better w/ plugins that allow you to set your max HP beyond the base game's 600.

moisterrific committed:
add examples for /buff

moisterrific committed:
forgot to add colours to /buff

moisterrific committed:
add color and examples for /gbuff

moisterrific committed:
Merge branch 'general-devel' into general-devel

moisterrific commented:
This doesn't cover all commands btw, I'll probably add those later.

hakusaro committed:
Merge branch 'general-devel' into general-devel
